### ISSUE TITLE
flags.png: Failed to load resource: the server responded with a status of 404 ()

### ISSUE THREAD

**Issue by bartdigitalpulse** [verbb/formie#1218]
### Describe the bug

When adding the phone number country dropdown field I get a Failed to load resource: the server responded with a status of 404 () in the console for the flags.png image. It expects to find it in cpresources/img/flags.png.

This should be the same path as the cdn? 

https://github.com/verbb/formie/blob/104c11117f247a21b6e615991c736bae3ec9ddfc/src/web/assets/frontend/dist/css/fields/phone-country.css#L919

### Steps to reproduce

1. Add phone number field with country code dropdown field
2. Load in frontend

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

Craft Pro 4.3.5

### Plugin version

	2.0.19

### Multi-site?

Yes

**Comment by engram-design** [comment]
We override that just a little further down in https://github.com/verbb/formie/blob/104c11117f247a21b6e615991c736bae3ec9ddfc/src/web/assets/frontend/dist/css/fields/phone-country.css#L960

The CSS you mention is pulled in from the library via https://github.com/verbb/formie/blob/104c11117f247a21b6e615991c736bae3ec9ddfc/src/web/assets/frontend/src/scss/fields/phone-country.scss#L1 so there's not much we can do about that instead of overriding it.

Are you doing any custom templating that might remove the following CSS selectors?

```
.field[data-type="verbb\\\\formie\\\\fields\\\\formfields\\\\Phone"], [data-field-type="phone-number"]
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by bartdigitalpulse** [comment]
@engram-design 
I no longer have the error. Thanks for the quick fix!

**Comment by engram-design** [comment]
Fixed in [2.0.20](https://github.com/verbb/formie/releases/tag/2.0.20)
